### PR TITLE
Final updates for 2.1

### PR DESCRIPTION
**Pull request by spboyer**
All builds pass for 2.1:

- from VSTS | [Build Report](https://msft-shayne.visualstudio.com/web/build.aspx?pcguid=45bdabab-1c01-40df-9c1b-0f35e2dea797&builduri=vstfs:%2f%2f%2fBuild%2fBuild%2f25&tracking_data=eyJTb3VyY2UiOiJFbWFpbCIsIlR5cGUiOiJOb3RpZmljYXRpb24iLCJTSUQiOiJtcy52c3MtYnVpbGQuYnVpbGQtcmVxdWVzdGVkLXBlcnNvbmFsLXN1YnNjcmlwdGlvbiIsIlNUeXBlIjoiQ09OIiwiUmVjaXAiOjEsIl94Y2kiOnsiTklEIjo0NzA5NTI4LCJNUmVjaXAiOiJtMD0xICIsIkFjdCI6ImRlMmU2NWQ2LTgzNzQtNDZlNC1hYzA4LTQyZDljZDMyNTEwMyJ9fQ==)
Manual Build of ASP.NET Workshop-ASP.NET Core-CI (ASP.NET Workshop)
Ran for 5.6 minutes (Hosted VS2017), completed at Wed 06/06/2018 03:08 AM

0 error(s), 0 warning(s)
--

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # ASP.NET Core - App Building Workshop
 
+![](https://msft-shayne.visualstudio.com/_apis/public/build/definitions/6bb14994-33a9-43d6-b737-dd68d09428c1/7/badge)
```
Comment (jongalloway) [inline README.md:3]: Badge points to https://msft-shayne.visualstudio.com - should this be the permanent location?

Comment (spboyer) [reply README.md:3]: Probably not. If we can get a .NET Foundation instance we can move the VSTS Build system there as well as the AppService Instances and ACR etc.

File: save-points/6-Deployment-docker/ConferencePlanner/docker-compose.override.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
-version: '3.4'
+version: '3.3'
```
Comment (jongalloway) [inline save-points/6-Deployment-docker/ConferencePlanner/docker-compose.override.yml:1]: Is there a reason to go from 3.4 to 3.3 here?

Comment (spboyer) [reply save-points/6-Deployment-docker/ConferencePlanner/docker-compose.override.yml:1]: VSTS Build Agent does not support 3.4 yet. No major difference.

File: README.md (hunk 1 above)
Comment (jongalloway) [reply README.md:3]: Okay, cool. Not a reason to block here, I'm merge and we can add another issue to track.

File: save-points/6-Deployment-docker/ConferencePlanner/docker-compose.override.yml (hunk 2 above)
Comment (jongalloway) [reply save-points/6-Deployment-docker/ConferencePlanner/docker-compose.override.yml:1]: Cool, makes sense.

**Pull request merged**